### DISCUSSION TITLE
尝试使用微软商店购买许可证时（微软商店）闪退？？？

### DISCUSSION THREAD

**Discussion by LittleNightSong** [WindowTop/WindowTop-App#364] [Q&A]
我在软件中点击“购买或登录（Microsoft Store）”时微软商店闪退，WindowTop提示购买未完成，可能已取消
![image](https://github.com/user-attachments/assets/910dbc7c-11be-4020-8e07-50ecf51628dd)
![image](https://github.com/user-attachments/assets/96eecb12-f8ef-4572-bf68-e255084662be)

**Comment by LittleNightSong** [comment]
图片2中的微软商店打开后（甚至没有显示界面）就闪退了

**Reply by gileli121** [reply]
Hello, I am aware about this problem. Sometimes it happens. Please try to re-login to Microsoft store, exit the app and try again. Start WindowTop from Microsoft store app (after relogin) If you still have the problem, try reinstalling the app. Let me know if it helps.

Another user has the same issue, and these instructions solved it for him. 

I am typing you my response also in your language using google translate:


-----------------------------------------


您好，我知道这个问题。有时会发生这种情况。请尝试重新登录 Microsoft 商店，退出应用程序并重试。从 Microsoft 商店应用程序启动 WindowTop（重新登录后）如果仍然有问题，请尝试重新安装应用程序。如果有帮助，请告诉我。

另一个用户有同样的问题，这些说明为他解决了问题。

我也使用谷歌翻译以您的语言输入我的回复：

**Reply by LittleNightSong** [reply]
感谢你的帮助，但是问题似乎并没有解决，我也会积极寻找更多的解决方案。
---
Thank you for your help, but the problem does not seem to be solved, and I will actively look for more solutions.

**Reply by gileli121** [reply]
@SongzqInChina 
Sorry for the late reply.
Please send email to contact@windowtop.info
and in the email include:
1. Link to here - https://github.com/WindowTop/WindowTop-App/discussions/364 
2. Purchase receipt from Microsoft Store or any document that proves that you purchased it

I will give you an activation key so you will be able to activate the app without using the Microsoft Store activation system.

-----

抱歉回复迟了。
请发送电子邮件至 contact@windowtop.info
并在电子邮件中包括：
1. 链接至此处 - https://github.com/WindowTop/WindowTop-App/discussions/364
2. Microsoft Store 的购买收据或任何证明您购买了它的文件

我会给您一个激活密钥，这样您就可以在不使用 Microsoft Store 激活系统的情况下激活该应用程序。

**Comment by wwxiaoqi** [comment]
这可能是 WIndows 操作系统的问题，请确定你不是在使用任何精简版，且正常安装了 Microsoft Store。

又或者是网络环境问题，当 Microsoft Store 尝试更新的时候下载更新安装包过程导致的问题，这个时候尝试更换 DNS 或者修改 hosts 的方式来使用它。

----来自中国朋友的建议。
